src/Features/Core/Portable/NavigateTo/INavigateToSearchService.cs

```diff
@@ -8,33 +8,83 @@
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis.Host;
 
-namespace Microsoft.CodeAnalysis.NavigateTo
+namespace Microsoft.CodeAnalysis.NavigateTo;
+
+internal interface INavigateToSearchService : ILanguageService
 {
-    internal interface INavigateToSearchService : ILanguageService
-    {
-        IImmutableSet<string> KindsProvided { get; }
-        bool CanFilter { get; }
+    IImmutableSet<string> KindsProvided { get; }
+    bool CanFilter { get; }
 
-        Task SearchDocumentAsync(Document document, string searchPattern, IImmutableSet<string> kinds, Document? activeDocument, Func<INavigateToSearchResult, Task> onResultFound, CancellationToken cancellationToken);
+    Task SearchDocumentAsync(
+        Document document,
+        string searchPattern,
+        IImmutableSet<string> kinds,
+        Func<INavigateToSearchResult, Task> onResultFound,
+        CancellationToken cancellationToken);
```

Comment:
the 'activeDocument' was not needed.  That guy is used for when there are hits in multiple docs, and we want to help order the hits so that hits in the activeDocument are prioritized (all else being equal).  but since we're only searching one doc, that's not ever relevant.